jspsych-uil-utils.js

```diff
@@ -191,4 +193,27 @@ var uil = {};
         return has_touchscreen
     }
 
+    /* ********* file loading *********** */
+
+    // The directory this script lives in
+    let scriptDir = document.currentScript.baseURI.split('/').slice(0, -1).join('/');
```

Comment:
I would like to see scriptDir as a constant. Unless we have good reason to change it.

Reply:
I thought about making it a constant, but I decided against it as it isn't a constant in the strictest definition.

A constant is well, always the same. This variable only exist _because_ it's value isn't constant, but calculated.

Reply:
I think from perspective and lifetime of the scripts/library the value should never change, hence be constant. It's calculated once and should never change. Hence, I'm in favor for const, since that communicates, the value doesn't change after initialization.